src/scripts/components/app/app.styl

```diff
@@ -5,6 +5,11 @@
   font-family: NotesEsa
   src: url('../../../../assets/fonts/NotesEsaReg.otf')
 
+@font-face
```

Comment:
do we still need this here? we only use the bold font in the markers which load the font by themselves, right?

Reply:
Isn't this the global font import for all components? We use both fonts in other components.

Reply:
Sorry i mean: you're right we don't use is yet 😄 But we need it at least for the splash screen title. I have used the regular one before, but that's not correct.

Reply:
ah I see 👍